meta/and.rb

```diff
@@ -1,4 +1,4 @@
-Mutant::Meta::Example.add do
+Mutant::Meta::Example.add :and do
```

Comment:
Yeah I agree with that style guideline. In this case I still think it would read better.